draft-ietf-mmusic-sdp-uks.md

```diff
@@ -273,50 +448,17 @@ conduct two sessions concurrently, if the attacker (Mallory) is on the network
 path between the victims, and if the same certificate - and therefore SDP
```

Comment:
> This attack creates an asymmetry in the beliefs about the identity of peers. However, this attack is only possible if the victim (Norma) is willing to conduct two sessions concurrently

Unless I'm mistaken, the precondition here is even more difficult than this phrasing would imply. Assuming my understanding is correct, I believe the following wording would be more accurate: `...is only possible if the victim (Norma) can be compelled to initiate two concurrent sessions nearly simultaneously...`

_Don't treat this comment as blocking. If you agree, make the change; if you don't, discard it and move the document forward._